README.md

```diff
@@ -14,12 +14,14 @@ MQTT Protocol library for Erlang/Elixir. This library provides the low level bui
 ### Example Erlang client usage
 
 ```erlang
-{:ok, Connection, false} = mqtt_client:connect(#{
+mqtt_client_sup:start_link(),
```

Comment:
This is actually more correct when used from the shell.
```suggestion
ok = application:start(mqtt),
```

The mqtt application is meant to be declared as a "dependency" in your .app (`applications` key) so it gets started automatically in your release.

Reply:
Fair enough.